### PR TITLE
Updated the top-level index file

### PR DESCRIPTION
**Pull request by alwasega**

### CODE REVIEW

**Comment by alwasega** [comment]
@sgibson91, @minrk, please check this PR in which I have updated the documentation's top-level index file.

File: docs/source/index.md [hunk 1]
```diff
@@ -46,41 +56,71 @@ JupyterHub also provides a {doc}`REST API <reference/rest-api>`.
 The JupyterHub team and Project Jupyter value our community, and JupyterHub
 follows the Jupyter [Community Guides](https://jupyter.readthedocs.io/en/latest/community/content-community.html).
 
-## Contents
+---
+
+## Documentation Contents
```
Comment (sgibson91) [inline docs/source/index.md:61]:
```suggestion
## Documentation Structure
```

I think this might be a more descriptive heading

**Review by sgibson91 — APPROVED** [review]
I have one suggestion, but otherwise LGTM!

File: docs/source/index.md (hunk 1 above)
Comment (alwasega) [reply docs/source/index.md]: Thanks @sgibson91. I have made the edit.

**Pull request merged**